src/services/sessions.js

```diff
@@ -2216,6 +2216,7 @@ module.exports = class SessionsHelper {
 	 */
 	static async bulkUpdateMentorNames(mentorsId, mentorsName) {
 		try {
+			mentorsId = mentorsId.toString()
```

Comment:
@param {Array} mentorsId

This states that mentorsId is an array, but you have applied toString(). Seems like a mismatch. Please clarify this. 

And if this is an array of mentor ids, rename this to mentorIds instead mentorsId.